source/guides/guides/continuous-integration.md

```diff
@@ -540,7 +540,11 @@ If you are not using one of the above CI providers then make sure your system ha
 ### Linux
 
 ```shell
+# Ubuntu
 apt-get install libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+
+# CentOS
+yum install -y xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver gnu-free-sans-fonts*
```

Comment:
`alsa-lib` also seems to be required, or else Cypress will not be able to get ahold of `libasound.so.2`.

The `gnu-free-sans-fonts*` seem to be optional, I was able to run Cypress without them and the resulting video does contain some nice-looking text.

```suggestion
yum install -y xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib
```